src/htcondor_es/convert_to_json.py

```diff
@@ -633,6 +639,9 @@ def convert_to_json(ad, cms=True, return_dict=False, reduce_data=False):
         ad["CommittedTime"] = ad["RemoteWallClockTime"]
     result["WallClockHr"] = ad.get("RemoteWallClockTime", 0)/3600.
 
+    if analysis and ad.get("JobStatus") == 2:
+        result["PilotRestLifeTimeMins"] = int((ad['MATCH_GLIDEIN_ToDie'] - ad['EnteredCurrentStatus'])/60)
```

Comment:
@todor-ivanov We should make sure this doesn't fail, so either put it inside a try/except clause or put a default value by using `get` instead of the direct lookup. I'm not sure what's better, it depends on how you want to handle the default values.